src/i18n/i18nConfig.ts

```diff
@@ -38,6 +38,7 @@ export const resources = {
     translation: pt_BR.translation,
     common: pt_BR.common,
     components: pt_BR.components,
+    permissions: pt_BR.permissions,
```

Comment:
Ah, nice catch! Would you mind extending all entries which are missing `permissions`? Thanks!

Reply:
Sure! I will extend this to other languages as well!

Reply:
@escapedcat Done!